### PR TITLE
[Modular] Resprites Latex (I'm sorry Sex 3 Devs)

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

Resprites the current Latex into something that looks more appealing. This isn't a direct port from Hyper Station. We have 5x more the latex sprites then them. But it's all being resprited based off the colors.
Before: 
image
After: 
image

_The Sprites are due to change in the future as I add/remove shading I like or dislike_

**Todo**

- [x] Male + Female Latex
- [x] I think I should touch up on the other clothes.
- [X] Overlay Tit Sprites
- [x] Things with four legs and snakes

**INGAME**

_This is an area where I show how the sprites look in game_
image

## How This Contributes To The Skyrat Roleplay Experience

Better latex is nice and I like the stuff. 

## Changelog
:cl: Syndishork
imageadd: Better Latex Sprites
/:cl:

### CODE REVIEW

**Comment by StrangeWeirdKitten** [comment]
This is what the sprites currently look in game. Note that I did use a couple extra items.

image

**Comment by StrangeWeirdKitten** [comment]
I think I got it all.

**Comment by StrangeWeirdKitten** [comment]
This PR is not stale 🔪

File: .vscode/launch.json [hunk 1]
```diff
@@ -7,14 +7,6 @@
 			"name": "Launch DreamSeeker",
 			"preLaunchTask": "Build All",
 			"dmb": "${workspaceFolder}/${command:CurrentDMB}"
-		},
```
Comment (Gandalf2k15) [inline .vscode/launch.json]: no

**Review by tf-4 — CHANGES_REQUESTED** [review]
Show ALL sprites in-game, please.

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/latex_catsuit.dm [hunk 2]
```diff
@@ -51,7 +51,7 @@
 		update_overlays()
 
 	//Breasts overlay for catsuit
-	if(B?.genital_size >= 6 || B?.genital_type == "pair")
+	if(B?.genital_size >= 6 & B?.genital_type == "pair")
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/latex_catsuit.dm:54]: Incorrect operator

**Comment by tf-4** [comment]
This still needs to have sprite pictures added.

**Comment by Gandalf2k15** [comment]
reopen with completed changes.

**Comment by chizzy376** [comment]
Any chances of trying to get this re-opened, @StrangeWeirdKitten ? I dig the new sprites you wanted to add, they look good

**Pull request closed without merging**